sync/upstream.py

```diff
@@ -469,6 +469,29 @@ def finish(self, status="complete"):
             else:
                 self.remote_branch = None
 
+    @property
+    def pr_head(self):
+        """
+        Retrieves the head of the PR ref: origin/pr/{pr_id}
+        :return: The SHA of the head commit.
+        """
+        if not self.pr:
+            logger.error("No PR ID found for %s" % self.process_name)
+            return
+
+        pr_ref = 'origin/pr/{}'.format(self.pr)
+
+        if pr_ref not in self.git_wpt.refs:
+            # Maybe we need to fetch?
+            self.git_wpt.remotes.origin.fetch()
```

Comment:
I think it's OK to just error here. In general we make sure we fetch before running any task rather than having random code fetch.

Reply:
makes sense, fixed